index.bs

```diff
@@ -80,7 +80,7 @@ it is likely that authorization requirements will necessitate it.
 This specification uses the terms "access token", "authorization server", "resource server" (RS), "token endpoint",
 "grant type", and "client" as defined by The OAuth 2.0 Authorization Framework [[!RFC6749]].
 
-Throughout this specification, we will use the term Identity Provider (IdP) in line with the
+Throughout this specification, we will use the term OIDC Provider (OP) in line with the
```

Comment:
In the spec this is generally referred to as `OpenID Provider (OP)` instead of `OIDC Provider`. Perhaps we should use the exact same terminology to prevent any confusion.